user-manual.md

```diff
@@ -0,0 +1,47 @@
+# A high level description of RecipeCart:
+This system is a website for uploading, searching, and rating recipes. Using our descriptive recipe tagging system (missing), users can specify dietary restrictions or preferences that match the recipes they wish to search for. Users can also leave ratings (missing) on recipes to let others know the average quality of the recipe.  
+
+# How to install the software:
```

Comment:
There should really only be a single h1. This and all following `#` should be `##` instead